### PR TITLE
OHRM5X-1436: Recruitment workflow action history connect BE

### PR DESCRIPTION
**Pull request by Super-Chama**

### CODE REVIEW

File: installer/Migration/V5_1_0/lang-string/recruitment.yaml [hunk 1]
```diff
@@ -69,7 +69,7 @@ langStrings:
     value: 'Job Offered'
     unitId: job_offered
   -
-    value: 'offer Declined'
```
Comment (SharukaNP) [inline installer/Migration/V5_1_0/lang-string/recruitment.yaml:72]: Good

File: src/client/src/orangehrmRecruitmentPlugin/pages/HireAction.vue [hunk 2]
```diff
@@ -94,10 +95,17 @@ export default {
...
         })
-        .then(() => {
+        .then(response => {
+          const {data} = response.data;
+          historyId = data.id;
           return this.$toast.updateSuccess();
         })
-        .then(() => this.onClickBack());
+        .then(() => {
+          navigate('/recruitment/candidate/{candidateId}/history/{historyId}', {
```
Comment (SharukaNP) [inline src/client/src/orangehrmRecruitmentPlugin/pages/HireAction.vue:104]: Great so we are redirecting to history after each action

**Review by SharukaNP — APPROVED** [review]

File: src/client/src/orangehrmRecruitmentPlugin/pages/ViewActionHistory.vue [hunk 3]
```diff
@@ -0,0 +1,351 @@
...
+      statuses: [
+        {id: 1, label: this.$t('recruitment.application_initiated')},
+        {id: 2, label: this.$t('recruitment.shortlisted')},
+        {id: 3, label: this.$t('recruitment.rejected')},
+        {id: 4, label: this.$t('recruitment.interview_scheduled')},
+        {id: 5, label: this.$t('recruitment.interview_passed')},
+        {id: 6, label: this.$t('recruitment.interview_failed')},
+        {id: 7, label: this.$t('recruitment.job_offered')},
+        {id: 8, label: this.$t('recruitment.offer_declined')},
+        {id: 9, label: this.$t('recruitment.hired')},
```
Comment (RajithaKumara) [inline src/client/src/orangehrmRecruitmentPlugin/pages/ViewActionHistory.vue:246]: @SharukaNP any idea

**Review by RajithaKumara — APPROVED** [review]

File: src/client/src/orangehrmRecruitmentPlugin/pages/ViewActionHistory.vue (hunk 3 above)
Comment (SharukaNP) [reply src/client/src/orangehrmRecruitmentPlugin/pages/ViewActionHistory.vue:246]: Here we don't call for that API for statuses right? It was for the drop down that I translated the data

**Pull request merged**